client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -130,7 +129,7 @@ class Upload extends Component {
 	}
 
 	redirectToWpAdmin() {
-		page( `https://${ this.props.siteSlug }/wp-admin/theme-install.php` );
+		window.location = `${ this.props.siteAdminUrl }theme-install.php`;
```

Comment:
Just a thought which came up: if this comes up more often do we need a general recommendation documented on how to navigate between Calypso and Atomic? Would it make sense to have something like this `redirectToWpAdmin` as a utility?